webapp/tests/test_finders.py

```diff
@@ -32,14 +32,16 @@ def setUp(self):
     def test_get_real_metric_path(self):
         real_wsp_file = os.path.join(self.tmp, "real.wsp")
         fake_wsp_file = os.path.join(self.tmp_2, "fake.wsp")
-        relative_wsp_path = "fake.wsp"
+        relative_wsp_path = "real.wsp"
```

Comment:
was this test failing? if not, why are we changing this?

Reply:
Yes the test was failing. The `os.symlink(fake_wsp_file, fake_wsp_file)` line seems to be odd as the source and destination for creating the symlink are same.
The `shutil.rmtree(self.tmp_2)
        os.remove("real.wsp")` lines were added a cleanup step.